django_requirements.txt

```diff
@@ -1,3 +1,3 @@
 # Install Django requirements, if we're using the optional Django-integrated
 # parts of XBlock
-Django >= 1.4, < 1.7
\ No newline at end of file
+Django >= 1.8, < 1.9
```

Comment:
Since this is a library intended for wide use, I would keep the requirement as broad as possible.  We can leave it as `>= 1.4, < 1.9`, can't we?